models/user.js

```diff
@@ -1,49 +1,27 @@
-const Joi = require("joi");
-const mongoose = require("mongoose");
+  
+const mongoose = require('mongoose');
 
-const User = mongoose.model(
-  "User",
-  new mongoose.Schema({
-    name: {
-      type: String,
-      required: true,
-      minlength: 5,
-      maxlength: 50
-    },
-    email: {
-      type: String,
-      required: true,
-      minlength: 5,
-      maxlength: 255,
-      unique: true
-    },
-    password: {
-      type: String,
-      required: true,
-      minlength: 5,
-      maxlength: 1024
-    }
-  })
-);
+const UserSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
```

Comment:
I think we should keep the character length restraints

Reply:
for name, and email only

Reply:
resolved, name restraint is 64 max and email restraint is 320 max